src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -412,15 +413,9 @@ def _flexible_server_params(command_group):
         with self.argument_context('{} flexible-server replica create'.format(command_group)) as c:
             c.argument('source_server', options_list=['--source-server'],
                        help='The name or resource ID of the source server to restore from.')
-            if command_group == 'mysql':
-                c.argument('tier', options_list=['--tier'],
-                           help='Compute tier of the server. Accepted values: Burstable, GeneralPurpose, Memory Optimized ')
-                c.argument('sku_name', options_list=['--sku-name'],
-                           help='The name of the compute SKU. Follows the convention'
-                                ' Standard_{VM name}. Examples: Standard_B1ms, Standard_D4s_v3 ')
-            if command_group == 'postgres':
-                c.argument('tier', options_list=['--tier'], validator=tier_validator,
-                           help='Compute tier of the server. Accepted values: Burstable, GeneralPurpose, Memory Optimized ')
+            c.ignore('location')
+            c.ignore('sku_name')
+            c.ignore('tier')
```

Comment:
The three arguments are totally removed in existing command?

Reply:
We are not using these three args for the command currently